src/test/createHierarchyRenderer.js

```diff
@@ -53,15 +55,15 @@ var React = require('React');
 function createHierarchyRenderer(...renderMethods) {
   var instances;
   var Components = renderMethods.reduceRight(
-    function(Components, renderMethod, depth) {
+    function(ComponentsAcc, renderMethod, depth) {
```

Comment:
What is Acc?  Accumulator?  Can we call it `ComponentsAccumulator`?  Just adding `Acc` really doesn't tell me much more than `Components` did.